config.yaml

```diff
@@ -64,10 +64,30 @@ params:
       # Should link to training and getting started pages
 
     - title: Community
-      url: /community/
+      sublinks:
+      - title: Contact
+        url: /community/
+      - title: Governance
+        url: /governance/
+      - title: Get Involved
+        url: /contribute/
+      - title: Roadmap
+        url: /roadmap/
+      - title: Google Summer of Code
+        url: /gsoc/
 
     - title: About Us
-      url: /about/
+      sublinks:
+      - title: About PyBaMM
+        url: /about/
+      - title: Ionworks
+        url: https://ion-works.com/
+        is_external: true
+      - title: PyBaMM on Twitter
+        url: https://twitter.com/pybamm_/
+        is_external: true
+      - title: PyBaMM Teams
+        url: /teams/
```

Comment:
I think the governance page should go under About Us

Reply:
Thanks, addressed in [9c27d34](https://github.com/pybamm-team/pybamm.org/pull/89/commits/9c27d34981539500714749cd223722a43124c7ab)